requirements.txt

```diff
@@ -30,6 +30,7 @@ invoke==2.2.0
 isort==5.12.0
 itsdangerous==2.1.2
 jaconv==0.3.4
+jieba==0.42.1
```

Comment:
Changes in this file should also be added to pyproject.toml

(I couldn't figure out how to do it better, so for now we're just adding it in the two places -- not great, but not terrible)